<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="BhuVarsh" failures="2" tests="2" name="Test(failed)" time="0.279" errors="0" timestamp="2022-11-16T17:56:31 IST">
  <testcase classname="testng.FailedTC" name="TC1" time="0.223">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method testng.FailedTC.TC1() didn&amp;apos;t finish within the time-out 200">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method testng.FailedTC.TC1() didn't finish within the time-out 200
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- TC1 -->
  <testcase classname="testng.FailedTC" name="TC3" time="0.009">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testng.FailedTC.TC3(FailedTC.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC3 -->
  <testcase classname="testng.FailedTC" name="TC5" time="0.0">
    <ignored/>
  </testcase> <!-- TC5 -->
  <testcase classname="testng.FailedTC" name="TC4" time="0.0">
    <ignored/>
  </testcase> <!-- TC4 -->
  <testcase classname="testng.FailedTC" name="TC2" time="0.0">
    <ignored/>
  </testcase> <!-- TC2 -->
</testsuite> <!-- Test(failed) -->
